src/unionfind.rs

```diff
@@ -92,6 +92,12 @@ impl<K> UnionFind<K>
         }
     }
 
+    /// Returns `true` if the given elements belong to the same set, and returns
```

Comment:
For future prs, just a small nit, prefer active language in the doc (So "Return...")